Features/MostPopular/MostPopularView.swift

```diff
@@ -89,9 +109,43 @@ class MostPopularView: UIView {
         super.init(coder: coder)
         setupUI()
     }
- 
-    private func setupUI() {
-        backgroundColor = UIColor(named: "backgroundColor")
+}
+
+// MARK: - Public functions
+extension MostPopularView {
+    func configureWith(name: String, score: Int, description: String?, status: WatchlistStatus?) {
```

Comment:
Looks niiiiice!

So, if I use it, I put it inside MostPopularView class?

Reply:
I'm a big fan of an organized namespace, so I'd like that 😄 
Additionally, if it's inside `MostPopularView` class you don't need this big name, simply `ViewConfiguration` would work, otherwise you need to declare it as `MostPopularView.MostPopularViewConfiguration` as opposed to `MostPopularView.ViewConfiguration`